addons/io_scene_gltf2/blender/com/gltf2_blender_material_helpers.py

```diff
@@ -22,23 +22,11 @@ def create_settings_group(name):
     gltf_node_group.inputs.new("NodeSocketFloat", "Occlusion")
     thicknessFactor  = gltf_node_group.inputs.new("NodeSocketFloat", "Thickness")
     thicknessFactor.default_value = 1.0
-    gltf_node_group.nodes.new('NodeGroupOutput')
-    gltf_node_group_input = gltf_node_group.nodes.new('NodeGroupInput')
-    gltf_node_group_input.location = -200, 0
-    return gltf_node_group
-
-def get_gltf_pbr_non_converted_name():
-    return "original glTF PBR data"
-
-def create_gltf_pbr_non_converted_group(name):
-    gltf_node_group = bpy.data.node_groups.new(name, 'ShaderNodeTree')
-
     specular = gltf_node_group.inputs.new("NodeSocketFloat", "specular glTF")
     specular.default_value = 1.0
     specularColor = gltf_node_group.inputs.new("NodeSocketColor", "specularColor glTF")
```

Comment:
Suggested naming, for consistency with other nodes —

```suggestion
    specular = gltf_node_group.inputs.new("NodeSocketFloat", "Specular")
    specular.default_value = 1.0
    specularColor = gltf_node_group.inputs.new("NodeSocketColor", "Specular Color")
```

I'm happy to make this change myself in a follow-up PR, if that would save you a bit of time.

Reply:
This is not as simple as simple naming change.
Current channel mapping during texture creation (see __get_image_data & __get_image_data_mapping) is based on socket name.
Because there is already a socket named "specular" on Principled shader, not sure we can have a second one. 
We need a way to differentiate the two ones, by checking socket.node.type for example

Reply:
Hmm I see – thank you! I'll think about this more and open a PR or issue if I can think of anything, but for now I'd be happy to go ahead with this PR. I think consolidating the sockets onto one node is a good UX improvement help already.

Reply:
I just pushed the name changes + changing checks to take the "same socket name on two different nodes" issue into account